### ISSUE TITLE
Warning when compiling

### ISSUE THREAD

**Issue by jtrees** [elridion/mete#2]
When compiling the dependencies for my project I get a warning.

```
==> mete
Compiling 7 files (.ex)
warning: unused import Mete.Protocol
  lib/mete/connection.ex:7

Generated mete app
```

```
# mix.exs

defmodule Foo.Mixfile do
  # ...
  defp deps do
    [
      {:mete, "~> 1.0.1"},
      # ...
    ]
  end
end
```

```
# mix.lock

%{
  "mete": {:hex, :mete, "1.0.1", "d76b5d03c329a545c121ae936decc7f0749d42b41162a9cb391fe78bcf8a09f2", [:mix], [], "hexpm", "dd9c22f0c86408accb3407908dd495da9b3bc3486aa6fffe55f73e55727a2e09"},
  # ...
}
```

```
$ elixir --version            
Erlang/OTP 21 [erts-10.0.5] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [hipe]

Elixir 1.10.4 (compiled with Erlang/OTP 21)
```

**Comment by elridion** [comment]
Was fixed in #3

**Comment by jtrees** [comment]
Thanks!
